applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -29,10 +29,30 @@ def __init__(self, model, custom_settings):
         # Set defaults and validate custom settings.
         self.dynamic_settings = KratosMultiphysics.Parameters("""
         {
-            "scheme_type"   : "bossak",
-            "damp_factor_m" :-0.3,
-            "rayleigh_alpha": 0.0,
-            "rayleigh_beta" : 0.0
+            "scheme_type"                : "bossak",
+            "damp_factor_m"              : -0.3,
+            "rayleigh_alpha"             : 0.0,
+            "rayleigh_beta"              : 0.0,
+            "determine_rayleigh_damping" : false,
+            "determine_rayleigh_damping_settings" : {
+                "echo_level"          : 0,
+                "write_on_properties" : true,
+                "damping_ratio_0"     : 0.0,
+                "damping_ratio_1"     : -1.0,
+                "eigen_system_settings" : {
+                    "solver_type"                : "FEASTSolver",
```

Comment:
we highly recommend to use the `eigen_eigensystem` from the EigenSolversApp for this
FEAST is way to sensitive regarding it's settings to be used in a general way, whereas the Eigen-Eigensolver is quite robust, and you can directly specify the number of EigenValues you want